docs/src/main/sphinx/release/release-478.md

```diff
@@ -0,0 +1,80 @@
+# Release 478 (22 Oct 2025)
+
+## General
+
+* Add support for column lineage in `UNNEST` clauses. ({issue}`16946`)
+* Add `retry-policy.allowed` configuration property to specify which query retry
+  policies can be selected by the user. ({issue}`26628`)
+* Add support for loading plugins from multiple directories. ({issue}`26855`)
+* Add the `/v1/integrations/gateway` endpoint that exposes additional cluster metrics for Trino Gateway.
+  ({issue}`26548`)
+* Allow dropping an uninitialized catalog that failed to load. ({issue}`26918`)
+* Improve performance of queries with an `ORDER BY` clause using `varchar` or `varbinary` types. ({issue}`26725`)
+* Improve performance of `MERGE` statements involving a `NOT MATCHED` case. ({issue}`26759`)
+* Improve performance of queries with joins which spill to disk. ({issue}`26076`)
```

Comment:
```
* Improve performance of queries involving `JOIN` when spilling is enabled. ({issue}`26076`)
```

Reply:
"when spilling is enabled" is not an accurate description, performance is improved only when the join actually spills to disk.
So this should rather be 
```
Improve performance of queries involving `JOIN` when the join spills to disk
```
 or something along those lines.

Reply:
Applied the latter version.